lib/src/simulations/rose_pattern.dart

```diff
@@ -58,6 +62,49 @@ class _RosePatternState extends State<RosePattern> {
         ),
         centerTitle: true,
       ),
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
+      floatingActionButton: Padding(
+        padding: EdgeInsets.all(8.0),
+        child: Visibility(
+          visible: animate,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: <Widget>[
+              FloatingActionButton(
+                heroTag: null,
+                backgroundColor: Colors.white,
+                child: (!animating)
+                    ? Icon(
+                  Icons.play_arrow,
+                  color: Colors.black,
+                )
+                    : Icon(
+                  Icons.pause,
+                  color: Colors.black,
+                ),
+                onPressed: (){
+                  setState(() {
+                    animating = !animating;
+                  });
+                },
+              )
+              ,FloatingActionButton(
+                heroTag: null,
+                backgroundColor: Colors.white,
+                child: Icon(
+                  Icons.highlight_off,
+                  color: Colors.black,
+                ),
+                onPressed: (){
+                  setState(() {
+                    globalKey.currentState.clearScreen();
+                  });
+                },
+              ),
+            ],
+          ),
+        ),
+      ),
       bottomNavigationBar: Container(
```

Comment:
This is an older simulation so we haven't implemented a `ListView` widget like in Lissajous Curve. Just change the Column child to ListView similarly to how it's been done in [Lissajous Curve](https://github.com/cod-ed/simulate/blob/master/lib/src/simulations/lissajous_curve.dart).